docs/docs/guides/api/export-and-import.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Export and import with ZITADEL
+---
+
+## Export from V1 to Import into V2
+
+To migrate from ZITADEL V1 to V2 the API provides you with a possibility to export all resources which are under your organizations.
+Currently, this doesn't include the following points:
+
+* Global policies
+* IAM members
+* Global IDPs
+* Global second/multi factors
+* Machine keys
+* PAT's
+* Application keys
+
+Which results in that if you want to import, and you have no defined organization-specific custom policies, the experience for your users will not be exactly like in your old instance.
+
```

Comment:
```suggestion
```suggestion

::note Note that the ressources will be migrated without the event stream. This means that you will not have the audit trail for the imported objects.

```
```